xml/cl.xml

```diff
@@ -5919,7 +5925,7 @@ server's OpenCL/api-docs repository.
                 <command name="clIcdSetPlatformDispatchDataKHR"/>
             </require>
         </extension>
-        <extension name="cl_loader_layers" revision="1.0.0" supported="opencl">
+        <extension name="cl_loader_layers" revision="2.0.0" supported="opencl">
```

Comment:
I'm not sure we're bumping the version correctly here, in the spec changes  `cl_loader_layers` goes to `1.0.1` and `cl_khr_icd` goes to `2.0.1`.

Could also add `require` tags for `clInitLayerWithProperties` and associated `CL_LAYER_PROPERTIES_LIST_END` & `cl_layer_properties`

Reply:
This should be addressed, let me know i missed something.